packages/examples/kitchen-sink/package.json

```diff
@@ -4,7 +4,7 @@
   "license": "MIT",
   "private": true,
   "scripts": {
-    "demo": "parcel src/index.html"
```

Comment:
Is this debugging?

Maybe we should standardize different names for builds and watch in examples — I find myself testing both.

Reply:
Would be a good idea, will revert it back to the original value for this PR